constraint/pkg/client/drivers/local/driver.go

```diff
@@ -30,6 +32,16 @@ import (
 const (
 	libRoot   = "data.lib"
 	violation = "violation"
+
+	templateRunTimeMillisName        = "templateRunTimeMillis"
+	templateRunTimeMillisDescription = "the number of milliseconds it took to evaluate all constraints for a template"
+
+	engineTypeName        = "engineType"
```

Comment:
This will collide with multi-engine:

https://github.com/maxsmythe/frameworks/blob/281f26c755af5da4c013e0b6eca3697807d655d3/constraint/pkg/client/drivers/rego/driver.go#L85-L88

A couple other changes also conflict. That being merged might be a blocker for this change.

Reply:
These consts could live out in a different package. Was there something in particular you wanted to suggest changing?